### DISCUSSION TITLE
Unable to increase the height of no results view and loading view in combobox

### DISCUSSION THREAD

**Discussion by RajaTheKing826** [adobe/react-spectrum#6634] [Q&A] [answered]
I want a custom no data view and loading view where my UI height is nearly **320PX** but in combobox I've observed that you are rendering the no results view as an item in list box , so the height is getting fixed to **32px** which is not expected in my case. What is the correct way to resolve this @snowystinger

**Comment by RajaTheKing826** [comment]
@snowystinger  @LFDanLu  any insights on this?

**Reply by LFDanLu** [reply, marked as answer]
I believe https://github.com/adobe/react-spectrum/blob/b46d23b9919eaec8ab1f621b52beced82e88b6ca/packages/%40react-spectrum/listbox/src/ListBoxBase.tsx#L58 is the relevant piece of code here. You'll have to modify the placeholder option passed to the ListLayout you are using in your listbox
